.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- Thank you for opening a PR! We really appreciate you taking the time to help out 🙌 -->
+
+#### What kind of changes does this PR include?
+<!-- Place an X in the [ ] for any of these that apply -->
+
+- [ ] Minor content fixes (broken links, typos, etc.)
+- [ ] New or updated content
+- [ ] Translated content
+- [ ] Changes to the docs site code
+- [ ] Something else!
+
+#### Description
+
+- What does this PR change? Give us a brief description.
+- Did you change something visual? A before/after screenshot can be helpful.
+- Is this change small enough not to impact translation work?
+  If so, please include `minor`, `broken link` or `typo` in the PR title
+- Is this a translation PR?  
+  - It’s helpful to title your PR something like “i18n(lang): PR description”
+  - Consider pinging your i18n-gang on Discord for a review.
+- Not on Discord? Join us at https://astro.build/chat
```

Comment:
Removing this more detailed advice so we don’t overload people. We should include it in a contributing guide when we have one. We can retitle PRs manually if necessary based on the answers to the checklist.

```suggestion
```